src/app/screens/Game/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import './styles.css';
+
+import Board from './components/Board';
+
+/* eslint-disable react/no-array-index-key */
+
+function calculateWinner(squares) {
+  const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+  for (let i = 0; i < lines.length; i += 1) {
+    const [a, b, c] = lines[i];
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      return squares[a];
+    }
+  }
+  return null;
+}
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      history: [
+        {
+          squares: Array(9).fill(null)
+        }
+      ],
+      stepNumber: 0,
+      xIsNext: true
+    };
+  }
+
+  handleClick(i) {
+    const history = this.state.history.slice(0, this.state.stepNumber + 1);
+    const current = history[history.length - 1];
```

Comment:
`const current = history.slice(-1);` to get the last item

Reply:
Had to add const current = history.slice(-1)[0]; because of object composition

Reply:
mm ok, try `history.slice().pop()`